app/controllers/meetings_controller.rb

```diff
@@ -83,7 +82,8 @@ def create
   def update
     if @meeting.update(meeting_params)
       @meeting_members = @meeting.members
-      notify_members(@meeting, @meeting_members, 'update_meeting')
+      send_notification(@meeting, @meeting_members, 'update_meeting')
+
 
       respond_to do |format|
         format.html { redirect_to meeting_path(@meeting.slug) }
```

Comment:
@julianguyen Is this the expected response `render json: @meeting.errors, status: :unprocessable_entity` in https://github.com/ifmeorg/ifme/pull/1080/files#diff-ec628dfcdda37a04a922f83384b41725L91

Reply:
Hmmm good question 🤔  I don't think so :/ 
I think you can actually get rid of that respond_to block and replace it with `redirect_to_path(meeting_path(@meeting.slug))`

Reply:
Does this apply to the other `respond to` blocks or this one specifically? @julianguyen

Reply:
Yeah, I would say so excluding the ones with alerts or notices.

Reply:
I think you can get rid of the respond block and format.json for redirectrs with alerts or notices and just do: `redirect_to(group_path(group_id), alert: t('meetings.leave.error'))` for instance.